provider/testdata/data/uprops/Bidi_C.toml

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
These need copyright headers.  Running `upropdump` with `-c` should add the ICU4C copyright header.  That might be sufficient, but the copyright tool will probably complain that you need to add the ICU4X copyright header as well.  I think we should probably skip the files in this directory in the tool, such that the ICU4C copyright header by itself is sufficient.

Reply:
I've added ICU4C copyright headers.